projects/plugins/jetpack/extensions/blocks/dialogue/components/timestamp-control.js

```diff
@@ -57,55 +59,78 @@ function setTimestampValue( typeValue, smh ) {
 	return smh.join( ':' );
 }
 
-function Timestamp( { value, className, onChange, shortLabel = false } ) {
+export function TimestampControl( {
+	value,
+	className,
+	onChange,
+	shortLabel = false,
+	skipForwardTime,
+	jumpBackTime,
+} ) {
 	const smh = value.split( ':' );
 	if ( smh.length <= 2 ) {
 		smh.unshift( '00' );
 	}
 
 	return (
-		<div className={ `${ className }__timestamp-controls` }>
-			<NumberControl
-				className={ `${ className }__timestamp-control__hour` }
-				label={ shortLabel ? __( 'Hour', 'jetpack' ) : __( 'Hour', 'jetpack' ) }
-				value={ smh[ 0 ] }
-				min={ 0 }
-				max={ 23 }
-				onChange={ hour => {
-					onChange( setTimestampValue( { hour }, smh ) );
-				} }
-			/>
-
-			<NumberControl
-				className={ `${ className }__timestamp-control__minute` }
-				label={
-					shortLabel ? _x( 'Min', 'Short for Minute', 'jetpack' ) : __( 'Minute', 'jetpack' )
-				}
-				value={ smh[ 1 ] }
-				min={ 0 }
-				max={ 59 }
-				onChange={ min => {
-					onChange( setTimestampValue( { min }, smh ) );
-				} }
-			/>
-
-			<NumberControl
-				className={ `${ className }__timestamp-control__second` }
-				label={
-					shortLabel ? _x( 'Sec', 'Short for Second', 'jetpack' ) : __( 'Second', 'jetpack' )
-				}
-				value={ smh[ 2 ] }
-				min={ 0 }
-				max={ 59 }
-				onChange={ sec => {
-					onChange( setTimestampValue( { sec }, smh ) );
-				} }
-			/>
+		<div className={ `${ className }__timestamp-container` }>
+			<div className={ `${ className }__timestamp-controls` }>
+				<NumberControl
+					className={ `${ className }__timestamp-control__hour` }
+					label={ shortLabel ? __( 'Hour', 'jetpack' ) : __( 'Hour', 'jetpack' ) }
+					value={ smh[ 0 ] }
+					min={ 0 }
+					max={ 23 }
+					onChange={ hour => {
+						onChange( setTimestampValue( { hour }, smh ) );
+					} }
+				/>
+
+				<NumberControl
+					className={ `${ className }__timestamp-control__minute` }
+					label={
+						shortLabel ? _x( 'Min', 'Short for Minute', 'jetpack' ) : __( 'Minute', 'jetpack' )
+					}
+					value={ smh[ 1 ] }
+					min={ 0 }
+					max={ 59 }
+					onChange={ min => {
+						onChange( setTimestampValue( { min }, smh ) );
+					} }
+				/>
+
+				<NumberControl
+					className={ `${ className }__timestamp-control__second` }
+					label={
+						shortLabel ? _x( 'Sec', 'Short for Second', 'jetpack' ) : __( 'Second', 'jetpack' )
+					}
+					value={ smh[ 2 ] }
+					min={ 0 }
+					max={ 59 }
+					onChange={ sec => {
+						onChange( setTimestampValue( { sec }, smh ) );
+					} }
+				/>
+			</div>
+
+			<div className={ `${ className }__timestamp-player` }>
+				<MediaPlayerControl
```

Comment:
Should this control also be in sync with the player?

Reply:
It syncs with the player globally, through the media-source store. `<MediaPlayerControl />` could work independently to other components.